### PR TITLE
PROD release 1.1.2

### LINKED ISSUE #48: PROD Release 1.1.2
Can we cut a new release for https://github.com/civisanalytics/muffnn/pull/46 and update `datascience-python`?

### PR DESCRIPTION
**Pull request by beckermr**
This PR updates the code for v1.1.2.

Closes #48

### CODE REVIEW

File: muffnn/mlp/tests/test_mlp_classifier.py [hunk 1]
```diff
@@ -261,3 +263,20 @@ def test_refitting():
...
+    X = StandardScaler().fit_transform(X)
+
+    mlp = MLPClassifier(n_epochs=10,
+                        solver_kwargs={'learning_rate': 0.05},
+                        random_state=4567).fit(X, y)
+
+    y_oos = cross_val_predict(mlp, X, y, cv=4, method='predict_proba')
+    auc = roc_auc_score(y, y_oos, average=None)
+
+    assert np.all(auc >= 0.96)
```
Comment (jmemich) [inline muffnn/mlp/tests/test_mlp_classifier.py]: Do you even care about this? This looks both brittle and unnecessary: you're really just performing a smoke test to see if you conform to the sklearn API.

Comment (beckermr) [inline muffnn/mlp/tests/test_mlp_classifier.py]: I do care. This is more of an integration test. I forgot to seed the CV iterator. Once everything is seeded, the test won't be brittle.
